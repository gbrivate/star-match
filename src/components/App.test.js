import React from 'react';

import renderer from 'react-test-renderer';
import StarMatch from './App';

describe('StarMatch', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<StarMatch />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
