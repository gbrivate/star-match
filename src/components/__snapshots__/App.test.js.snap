// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <h1>
    TEST
  </h1>
  This is a sample stateful and server-side rendered React application.
  <br />
  <br />
  Here is a button that will track how many times you click it:
  <br />
  <br />
  <button
    onClick={[Function]}
  >
    0
  </button>
</div>
`;

exports[`StarMatch renders correctly 1`] = `
<div
  className="game"
>
  <div
    className="help"
  >
    Pick 1 or more numbers that sum to the number of stars
  </div>
  <div
    className="body"
  >
    <div
      className="left"
    >
      <div>
        <div
          className="star"
        />
        <div
          className="star"
        />
        <div
          className="star"
        />
        <div
          className="star"
        />
        <div
          className="star"
        />
      </div>
    </div>
    <div
      className="right"
    >
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        1
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        2
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        3
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        4
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        5
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        6
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        7
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        8
      </button>
      <button
        className="number"
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "lightgray",
          }
        }
      >
        9
      </button>
    </div>
  </div>
  <div
    className="timer"
  >
    Time Remaining: 
    10
  </div>
</div>
`;
